src/h5phelpers/forms/interactionForm.js

```diff
@@ -34,14 +50,45 @@ export const createInteractionForm = (field, params, wrapper, parent) => {
     '.h5peditor-copypaste-wrap',
   ];
 
+  const interactionDataWrapper = wrapper.querySelector('.field.field-name-label');
+  const hiddenInteractionDataSelectors = [
+    '.field-name-showAsOpenSceneContent',
+  ];
+
+  //Remove fields in that we don't want to show when library is not AdvancedText
+  if(!params.action.library.includes("H5P.AdvancedText")) {
+    removeElements(interactionDataWrapper, hiddenInteractionDataSelectors, '')
+  } else {
```

Comment:
Please follow the same `if` `else` block style already used in this project for consistency.